core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestPushJoinIntoTableScan.java

```diff
@@ -221,44 +219,6 @@ public static Object[][] testPushJoinIntoTableScanParams()
         };
     }
 
-    @Test
-    public void testPushJoinIntoTableScanDoesNotTriggerWithUnsupportedFilter()
```

Comment:
consider converting this test to one that validates the pushdown actually happenes (so we'll have a test for non-trivial `ConnectorExpression` pushdown)

Reply:
restored